playwright-tests/pages/index.spec.ts

```diff
@@ -35,7 +35,7 @@ test('Homepage links are correct', async ({ page }) => {
     const allTechLinks = page.locator('a:has-text("View All Technologies")');
     const allProductsLinks = page.locator('a:has-text("View All Products")');
     // There are mobile and desktop links rendered, check both links
-    await allLinksHaveHref(allLangLinks, '/developer/languages/');
-    await allLinksHaveHref(allTechLinks, '/developer/technologies/');
-    await allLinksHaveHref(allProductsLinks, '/developer/products/');
+    await allLinksHaveHref(allLangLinks, '/developer/languages');
+    await allLinksHaveHref(allTechLinks, '/developer/technologies');
+    await allLinksHaveHref(allProductsLinks, '/developer/products');
```

Comment:
Just curious why the reason for this change?

Reply:
after removing the `thumbnailLoader` from all of the `Image` components, a lot of them were giving 404s. i figured out that this has to do with an unnecessary trailing slash that was being added in a util file here:

[https://github.com/mongodb/devcenter/pull/305/files#diff-02af2aabafccca1d0923edaea39e1c8e63813a2e8097f123795ccb850e876ab6L25](https://github.com/mongodb/devcenter/pull/305/files#diff-02af2aabafccca1d0923edaea39e1c8e63813a2e8097f123795ccb850e876ab6L25)

this change fixed the images and allowed them to load, but the tests all had hardcoded trailing slashes when checking the hrefs, so that's why i removed them from all of the related tests

Reply:
Rather than updating this to remove the `/` here, can we modify the utils below so that:
```
    if (urlPath[urlPath.length - 1] !== '/' && !urlPath.startsWith('_next/images')) {
        urlPath += '/';
    }
```
Or whatever `!urlPath.startsWith('_next/images')` should be so that the images don't end up being 404s?